platform/util/platform-util/src/main/java/org/codice/ddf/platform/util/TemporaryFileBackedOutputStream.java

```diff
@@ -83,12 +101,30 @@ public void close() throws IOException {
         }
         try {
             fileBackedOutputStream.close();
-            fileBackedOutputStream.reset();
+            reset();
         } finally {
             isClosed = true;
         }
     }
 
+    /**
+     * Reset fileBackedOutputStream and retry if it fails.
+     */
+    @SuppressWarnings("unchecked")
+    private void reset() {
+
+        RetryPolicy retryPolicy = new RetryPolicy().retryOn(IOException.class)
+                .withBackoff(INITIAL_RETRY_SLEEP, MAX_DELAY, INITIAL_RETRY_SLEEP_UNIT)
+                .withMaxRetries(MAX_RETRY_ATTEMPTS);
+
+        Failsafe.with(retryPolicy)
+                .onFailedAttempt(throwable -> LOGGER.debug("failed to reset FBOS, will retry",
+                        throwable))
+                .onFailure(throwable -> LOGGER.debug("failed to reset FBOS", throwable))
```

Comment:
if the filename is in the exception then that should be enough information for the administrator.  I'd state something like "Failed to delete temporary file: <exception>" since and admin won't have any idea what an FBOS is. Otherwise looks good 👍